src/gleam_community/ansi.gleam

```diff
@@ -2342,11 +2342,11 @@ pub fn bg_colour(text: String, colour: Colour) -> String {
 /// </div>
 ///
 pub fn strip(text: String) -> String {
-  let regex_options = regex.Options(False, True)
-  let assert Ok(r) = regex.compile("(?:\\[(?:\\d+;?)+m)+", with: regex_options)
+  let regexp_options = regexp.Options(False, True)
```

Comment:
I know it was like this before, but I really wish this has the `case_insensitive` and `multi_line` labels on this constructor